site/install.md

```diff
@@ -1,15 +1,556 @@
 # Create and manage the operator
 
-An operator is an application-specific controller that extends Kubernetes to create, configure, and manage instances 
```

Comment:
It looks like the information here that was moved to 'user-guide' could also be copied to the Overview/Intro section of the new 'install.md' (the former helm-charts.md), and would make its introduction nicer...   E.g. it'd be helpful if the introduction to 'create and manage the operator' described what the operator was in the first place.

Reply:
Sure, I'll put some of the information into the Overview section.

Reply:
Actually, it's going into the Intro section, prior to the Overview.